README.markdown

```diff
@@ -261,6 +261,10 @@ protoc --plugin=node_modules/ts-proto/protoc-gen-ts_proto ./batching.proto -I.
 
   This is also useful if you want "only types".
 
+- With `--ts_proto_opt=outputPartialMethods=false`, the `Message.fromPartial` method for working with partial JSON-coded data will not be output.
+
+  Also, if `outputJsonMethods=false` is provided this method will not be generated.
```

Comment:
I know historically this was the case, but maybe we should just fully decouple `outputJsonMethods` and `outputPartialMethods` now that you've added this 2nd flag.

Maybe just remove this line, and let's have `outputJsonMethods` be completely separate from `outputPartialMethods`.

This will be a breaking change b/c users will have to also start setting `outputPartialMethods=false` in addition to `outputJsonMethods=false` but I think that's fine.

Reply:
I agree this should be decoupled, done